### ISSUE TITLE
Windows version is slow

### ISSUE THREAD

**Issue by JayFoxRox** [espes/xqemu#75]
There's a bug with timers on Windows that makes it run much slower than it should.

---

Nobody cared because none of the current contributors use Windows.
It has been known for years that there is a bug. From ["Getting Started" (since August 2015)](https://github.com/espes/xqemu/wiki/Getting-Started/a1b67754bc964c774e52d20590b640c0af85b2de):

> There's a bug with timers on Windows that makes it run much slower than it should. YOU should fix it for me.

If the developers can't reproduce the issue or are not affected by it, they can't / won't fix it.
This is an open-source project, so we depend on contributions by others ("YOU"). In particular we need people to fix stuff they care about. If someone cares about Windows, they should work on Windows support.
**tl;dr: We need someone on Windows who debugs and fixes the issue.**

**Comment by zaykho** [comment]
^ This, this was what I was talking about month ago in the XQEMU IRC channel, but nobody cared.  :- |

Even with a spec like this:

i7 6700k 4.0 Ghz (4.8 Ghz OC)
64GB (4x16) DDR4 2133 Mhz Corsair
Asus ROG STRIX - GTX 1080, 1835 MHz, 8GB
Samsung 960 PRO NVMe M.2, 512GB
Windows 7 Pro x64

It take 40 minute to smashdrive to reach the title menu. And for some reasons, if I start multiple XQEMU instances, and play the same game, the later instance get faster (reaching same state in less than 2 minutes.......)

Old version of CXBX-reloaded have this problem too, but now it's seems to be fixed when we activate : Debug Output ( GUI ) and ( Kernel ) both to console, doing so, make the emulator to run at normal speed.

Without that, in the old CXBX-reloaded, when trying to load Quantum redshift, I have found that, if you just let the emulator go, the speed is extremely slow, but as soon as you click on the console Kernel window and then move and click inside the CXBX window (main game) the speed goes back to 60 FPS for couple of seconds (video and sound goes normal during that time) and the kernel window message update faster too, and after 2/3 seconds, it goes back to 1 FPS.  

Recent version of CXBX fixed that.

I don't know what it's causing this, but XQEMU was unusable for me because of this bug.

I can provide a video or a gift later to show how this problem occur.

**Comment by WLF0X** [comment]
Perhaps because the second time it runs it has already created all the necessary files from being run the first time?

**Comment by JayFoxRox** [comment]
XQEMU does not create any files.

Multiple instances running faster is probably related to poor system administration. Try thread pinning or setting performance modes of your CPU.

This *could* also be the bug in the Windows portion of QEMU (that we speak about here). Again: it's not that nobody cares; it's that nobody is willing to look into / fix the issue. XQEMU does not have any active developers who run Windows as their main OS. If Windows users want to see this issue fixed, they'll have to debug (and fix) those issues themselves. This has been addressed in the first post, too.

We don't really care about CxBx bugs / stories here (unless they are actually emulation related; which this issue isn't).

---

**Now, back to topic please**: There is a bug and we don't need any "This affects me too!". We need "I've looked into this: this is why" and "Here is a fix"
